File: src/Java/Janitor Troubles/JanitorTroubles.java
public class JanitorTroubles {
    public static void main(String[] args) {
        Kattio io = new Kattio(System.in);
        int s1 = io.getInt(), s2 = io.getInt(), s3 = io.getInt(), s4 = io.getInt();
        double semiperimeter = (s1 + s2 + s3 + s4) / 2.0;
        io.print(Math.sqrt((semiperimeter - s1) * (semiperimeter - s2)
                * (semiperimeter - s3) * (semiperimeter - s4)));
        io.flush();
    }
}

File: src/Python 3/Janitor Troubles/JanitorTroubles.py
import math

arr = [int(i) for i in input().split()]
s = sum(arr) / 2 # find semiperimeter
print(math.sqrt((s - arr[0]) * (s - arr[1]) * (s- arr[2]) * (s - arr[3]))) # Brahmagupta's formula